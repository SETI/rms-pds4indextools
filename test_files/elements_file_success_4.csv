/pds:Product_Observational/pds:Identification_Area[1]/pds:logical_identifier[1],/pds:Product_Observational/pds:Identification_Area[1]/pds:version_id[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Time_Coordinates[1]/pds:start_date_time[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Time_Coordinates[1]/pds:stop_date_time[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Primary_Result_Summary[1]/pds:Science_Facets[1]/pds:wavelength_range[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Primary_Result_Summary[1]/pds:Science_Facets[2]/pds:wavelength_range[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Investigation_Area[1]/pds:Internal_Reference[1]/pds:lid_reference[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Observing_System[1]/pds:Observing_System_Component[1]/pds:Internal_Reference[1]/pds:lid_reference[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Observing_System[1]/pds:Observing_System_Component[2]/pds:Internal_Reference[1]/pds:lid_reference[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Observing_System[1]/pds:Observing_System_Component[3]/pds:Internal_Reference[1]/pds:lid_reference[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Target_Identification[1]/pds:Internal_Reference[1]/pds:lid_reference[1],/pds:Product_Observational/pds:Observation_Area[1]/pds:Target_Identification[2]/pds:Internal_Reference[1]/pds:lid_reference[1],/pds:Product_Observational/pds:Reference_List[1]/pds:Internal_Reference[1]/pds:lid_reference[1],/pds:Product_Observational/pds:Reference_List[1]/pds:Internal_Reference[2]/pds:lid_reference[1],/pds:Product_Observational/pds:Reference_List[1]/pds:Internal_Reference[3]/pds:lid_reference[1],/pds:Product_Observational/pds:Reference_List[1]/pds:Internal_Reference[4]/pds:lid_reference[1],/pds:Product_Observational/pds:Reference_List[1]/pds:Internal_Reference[5]/pds:lid_reference[1],/pds:Product_Observational/pds:Reference_List[1]/pds:Internal_Reference[6]/pds:lid_reference[1],/pds:Product_Observational/pds:Reference_List[1]/pds:Internal_Reference[7]/pds:lid_reference[1],/pds:Product_Observational/pds:Identification_Area[1]/pds:Modification_History[1]/pds:Modification_Detail/pds:version_id[1]
urn:nasa:pds:uranus_occ_u149_irtf_320cm:data:2200nm_counts-v-time_occult,1.0,1998-11-06T04:38:18.7028Z,1998-11-06T05:52:29.0559Z,Infrared,Infrared,urn:nasa:pds:context:investigation:observing_campaign.earth-based-uranus-stellar-occultations,urn:nasa:pds:context:facility:observatory.irtf-maunakea,urn:nasa:pds:context:telescope:irtf-maunakea.3m2,urn:nasa:pds:context:instrument:multi-host.generic_insp_hsp,urn:nasa:pds:context:target:planet.uranus,urn:nasa:pds:context:target:ring.uranus.rings,urn:nasa:pds:uranus_occ_support:document:earth-based-uranus-stellar-occultations-user-guide,urn:nasa:pds:uranus_occ_u149_irtf_320cm:browse:2200nm_obs_geom,urn:nasa:pds:uranus_occ_u149_irtf_320cm:browse:2200nm_earth,urn:nasa:pds:uranus_occ_u149_irtf_320cm:browse:2200nm_alt,urn:nasa:pds:uranus_occ_u149_irtf_320cm:browse:2200nm_counts-v-time_occult,urn:nasa:pds:uranus_occ_u149_irtf_320cm:browse:2200nm_counts-v-time_normalization,urn:nasa:pds:uranus_occ_support,NaN
urn:nasa:pds:cassini_iss_cruise:data_raw:1357539630n,1.0,2001-01-07T06:08:47.793Z,2001-01-07T06:08:48.013Z,Near Infrared,NaN,urn:nasa:pds:context:investigation:mission.cassini-huygens,urn:nasa:pds:context:instrument_host:spacecraft.co,urn:nasa:pds:context:instrument:issna.co,NaN,urn:nasa:pds:context:target:calibration_field.dark_sky,NaN,urn:nasa:pds:cassini_iss_cruise:data_raw:index,urn:nasa:pds:cassini_iss_cruise:data_raw:body-geometry,urn:nasa:pds:cassini_iss_cruise:body-inventory,urn:nasa:pds:cassini_iss_cruise:ring-geometry,urn:nasa:pds:cassini_iss_saturn:document:iss-data-user-guide,urn:nasa:pds:cassini_iss_cruise:browse_raw:1357539630n-full,NaN,1.0
