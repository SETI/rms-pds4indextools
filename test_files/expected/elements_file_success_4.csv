pds:Product_Observational/pds:Observation_Area<1>/pds:comment<1>,pds:Product_Observational/pds:Reference_List<1>/pds:Internal_Reference<1>/pds:comment<1>,pds:Product_Observational/pds:Reference_List<1>/pds:Internal_Reference<2>/pds:comment<1>,pds:Product_Observational/pds:Reference_List<1>/pds:Internal_Reference<3>/pds:comment<1>,pds:Product_Observational/pds:Reference_List<1>/pds:Internal_Reference<4>/pds:comment<1>,pds:Product_Observational/pds:Reference_List<1>/pds:Internal_Reference<5>/pds:comment<1>,pds:Product_Observational/pds:Reference_List<1>/pds:Internal_Reference<6>/pds:comment<1>,pds:Product_Observational/pds:Reference_List<1>/pds:Internal_Reference<7>/pds:comment<1>,pds:Product_Observational/pds:Reference_List<1>/pds:Internal_Reference<8>/pds:comment<1>
NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN,NaN
"This data file was migrated from the original PDS3 file: COISS_1004/data/1357539422_1357559280/N1357539630_1.IMG The content of this data file was reorganized in order to make it PDS4 compliant. Specifically, prefix bytes were removed from each line of the image and appended at the end. The resulting data file remains compatible with the VICAR standard. Values for instrument attributes have been derived from the corresponding values in the original PDS3 label.",The index table of basic parameters associated with each observation.,The index table of body surface geometry information associated with each observation.,An index listing every Saturn system body expected to fall within each field of view.,The index table of ring geometry information associated with each observation.,The ISS Data User's Guide,The browse image in JPEG format that illustrates the scientific content of this image.,NaN,NaN
NaN,The User Guide for Earth-based Uranus Stellar Occultations.,Diagram of the Uranus system showing the occultation track.,Diagram of the view of the Earth from Uranus at mid-occultation Time.,Plot of the altitude (in degrees) of Uranus and the sun relative to the horizon over the duration of the occulation.,Low resolution plot of counts vs. time for the entire occultation.,Illustrates some of the processing steps involved in normalizing a time-series occultation light curve from counts per second to units of stellar flux.,The fitted model of the ring system upon which the data are based.,The support bundle for all of the Earth-based Uranus system occultations.
